### PR TITLE
docs: translate getting-started

### PR DESCRIPTION
**Pull request by zhangyuthink**
### Description

translate getting-started

### Checklist

- [X] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [X] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: docs/zh/latest/getting-started/README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+## 前置条件
+
+快速启动脚本需要以下条件：
+
+* 已安装 [Docker](https://docs.docker.com/get-docker/)，用于部署  **etcd** 和 **APISIX**。
+* 已安装 [curl](https://curl.se/)，用于验证 APISIX 是否安装成功。
+
+## 安装 APISIX
+
+:::注意
```
Comment (xuruidong) [inline docs/zh/latest/getting-started/README.md:27]: `:::caution` or `:::caution 注意`

File: docs/zh/latest/getting-started/key-authentication.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+### 前置条件
+
+1. 参考[快入门指南](./README.md)完成 APISIX 的安装。
+2. 完成[配置路由](./configure-routes.md#route-是什么)。
+
+### 创建消费者
+
+创建一个名为 `tom` 的消费者，并启用 `key-auth` 插件，密钥设置为 `secret-key`。所有携带密钥 `secret-key` 的请求都会被识别为消费者 `tom`。
+
+:::警告
```
Comment (xuruidong) [inline docs/zh/latest/getting-started/key-authentication.md:58]: ditto，
refer to  https://docusaurus.io/docs/next/markdown-features/admonitions

File: docs/zh/latest/getting-started/configure-routes.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+1. 参考[入门指南](./README.md)完成 APISIX 的安装。
+
+## 创建路由
+
+你可以创建一个路由，将客户端的请求转发至 [httpbin.org](http://httpbin.org)（这个网站能测试 HTTP 请求和响应的各种信息）。
+
+通过下面的命令，你将创建一个路由，把请求`http://127.0.0.1:9080/ip` 转发至 [httpbin.org/ip](http://httpbin.org/ip)：
+
+[//]: <TODO: Add the link to the authorization of Admin API>
```
Comment (kayx23) [inline docs/zh/latest/getting-started/configure-routes.md:39]: Please remove all these `[//]: <TODO: xxx>`. It was copied over initially have doesn't have much to do with apache/apisix docs.

File: docs/zh/latest/getting-started.md [hunk 4]
```diff
@@ -1,249 +0,0 @@
...
-
-我们已经创建了路由与上游服务，现在可以通过以下命令访问上游服务：
-
-```bash
-curl -i -X GET "http://127.0.0.1:9080/anything/foo?arg=10" -H "Host: example.com"
-```
-
-该请求将被 APISIX 转发到 `http://httpbin.org:80/anything/foo?arg=10`。
-
-## 使用 APISIX Dashboard
```
Comment (kayx23) [inline docs/zh/latest/getting-started.md:227]: Not sure if we should remove this doc completely. The getting start docs built by API7 would not mention anything about dashboard.

File: docs/zh/latest/getting-started/key-authentication.md [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+### 前置条件
+
+1. 参考[快入门指南](./README.md)完成 APISIX 的安装。
+2. 完成[配置路由](./configure-routes.md#route-是什么)。
+
+### 创建消费者
+
+创建一个名为 `tom` 的消费者，并启用 `key-auth` 插件，密钥设置为 `secret-key`。所有携带密钥 `secret-key` 的请求都会被识别为消费者 `tom`。
+
+:::caution 警告
```
Comment (kayx23) [inline docs/zh/latest/getting-started/key-authentication.md:58]:
```suggestion
:::caution
```

File: docs/zh/latest/getting-started.md (hunk 4 above)
Comment (zhangyuthink) [reply docs/zh/latest/getting-started.md:227]: Can we add apisix-dashboard.md in /getting-started?

File: docs/zh/latest/getting-started/key-authentication.md [hunk 6]
```diff
@@ -0,0 +1,183 @@
...
+</head>
+
+> 本教程由 [API7.ai](https://api7.ai/) 编写。
+
+API 网关主要作用是连接 API 消费者和提供者。出于安全考虑，在访问内部资源之前，应先对消费者进行身份验证和授权。
+
+![身份验证](https://static.apiseven.com/uploads/2023/02/08/8mRaK3v1_consumer.png)
+
+APISIX 拥有灵活的插件扩展系统，目前有很多可用于用户身份验证和授权的插件。例如：
+
```
Comment (kayx23) [inline docs/zh/latest/getting-started/key-authentication.md:17]: Could you update the following links to their corresponding Chinese pages?

File: docs/zh/latest/getting-started/key-authentication.md [hunk 7]
```diff
@@ -0,0 +1,183 @@
...
+- [Key Authentication](https://apisix.apache.org/docs/apisix/plugins/key-auth/)
```
Comment (kayx23) [inline docs/zh/latest/getting-started/key-authentication.md:18]: Could you update the following links to their corresponding Chinese pages?

File: docs/zh/latest/getting-started/load-balancing.md [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+
+1. 将 `pass_host` 字段设置为 `node`，将传递请求头给上游。
+2. 将 `scheme` 字段设置为 `https`，向上游发送请求时将启用 TLS。
+
+:::
+
+## 验证
+
+这两个服务返回不同的数据。
+The two services respond with different data.
```
Comment (kayx23) [inline docs/zh/latest/getting-started/load-balancing.md:56]:
```suggestion
```

**Pull request merged**